usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -80,13 +81,13 @@ def signal_handler(signal, frame):
         elif len(submission_data) > 1:
             raise CommandError('Found multiple submissions with id ' + str(submission_id))
 
-        submission_data = submission_data[0]
+        submission_data = submission_data[0].copy()
         broker_submission_id = submission_data['submission_id']
         del submission_data['submission_id']  # We use broker_submission_id, submission_id is our own PK
         submission_attributes = get_submission_attributes(broker_submission_id, submission_data)
 
         logger.info('Getting File A data')
-        db_cursor.execute('SELECT * FROM appropriation WHERE submission_id = %s', [submission_id])
+        db_cursor.execute('SELECT * FROM appropriation WHERE submission_id = %s' % submission_id)
```

Comment:
Isn't the original the best practice to prevent sql injection? Why change it?

Reply:
This results from this PR being branched from #1333. The reason it was done like that was because the `PhonyCursor` wasn't able to process the query for the integration tests. I can look into modifying `PhonyCursor` to support arguments to prevent SQL injection.

Reply:
@tony-sappe this has been resolved in the other PR